assets/blocks/quiz/quiz-block/pagination-settings.js

```diff
@@ -87,16 +86,11 @@ const QuestionsControl = ( { settings, onChange, ...props } ) => {
 /**
  * Quiz sidebar settings.
  *
- * @param {Object}   props               Component props.
- * @param {Object}   props.settings      Pagination settings object.
- * @param {Function} props.onChange      Callback called when a setting changed.
- * @param {number}   props.questionCount Number of questions in the quiz.
+ * @param {Object}   props          Component props.
+ * @param {Object}   props.settings Pagination settings object.
+ * @param {Function} props.onChange Callback called when a setting changed.
  */
-export const PaginationSidebarSettings = ( {
-	settings,
-	onChange,
-	questionCount,
```

Comment:
The reason to remove the `questionCount` was that it was generating a JS error and it wasn't used in the pagination settings logic.